src/js/consts/reasonCodes.js

```diff
@@ -11,6 +11,17 @@ const reasonCodes = {
   RETURNED: 'Returned',
   SCRAPPED: 'Scrapped',
   STOLEN: 'Stolen',
+  BIN_CORRECTION: 'Bin correction',
+  LOT_CORRECTION: 'Lot correction',
+  INCORRECT_LOT_PICKED: 'Incorrect lot picked',
+  FIX_NEGATIVE_INVENTORY: 'Fix negative inventory',
+  INBOUND_NOT_RECORDED: 'Inbound not recorded',
+  OUTBOUND_NOT_RECORDED: 'Outbound not recorded',
+  RETURN_NOT_RECORDED: 'Return not recorded',
+  ERROR_IN_INBOUND: 'Error in inbound',
+  ERROR_IN_OUTBOUND: 'Error in outbound',
+  BIN_LOCATION_MISSING: 'Bin location missing',
+  MISPLACED: 'Misplaced',
   OTHER: 'Other',
```

Comment:
I had two thoughts while making this change:
1) This const is named "reasonCodes" but it only has cycle count reason codes. We should consider renaming it for clarity
2) Do we even need this list? The CC resolution flow fetches this list from the backend via API so hopefully we can do the same for reporting. That way we can get properly translated reason codes too.

I'll leave that refactor for after the release though